File: prog_python/OperacionesBasicas.py
# Función para realizar la suma de los dos primeros números y la multiplicación de los dos últimos
def operaciones_basicas(a, b, c, d):
    suma = a + b
    producto = c * d
    return suma, producto

# Solicitar los números al usuario
num1 = int(input("Ingrese el primer número: "))
num2 = int(input("Ingrese el segundo número: "))
num3 = int(input("Ingrese el tercer número: "))
num4 = int(input("Ingrese el cuarto número: "))

# Llamar a la función y obtener los resultados
resultado_suma, resultado_producto = operaciones_basicas(num1, num2, num3, num4)

# Mostrar los resultados
print(f"La suma de los dos primeros números es: {resultado_suma}")
print(f"El producto de los dos últimos números es: {resultado_producto}")

File: prog_java/OperacionesBasicas.java
package prog_java;

    import java.util.Scanner;

public class OperacionesBasicas {
    public static void main(String[] args) {
        // Crear un objeto Scanner para recibir la entrada del usuario
        Scanner scanner = new Scanner(System.in);

        // Solicitar los cuatro números enteros al usuario
        System.out.println("Ingrese el primer número:");
        int num1 = scanner.nextInt();

        System.out.println("Ingrese el segundo número:");
        int num2 = scanner.nextInt();

        System.out.println("Ingrese el tercer número:");
        int num3 = scanner.nextInt();

        System.out.println("Ingrese el cuarto número:");
        int num4 = scanner.nextInt();

        // Realizar la suma de los dos primeros números
        int suma = num1 + num2;

        // Realizar el producto de los dos últimos números
        int producto = num3 * num4;

        // Mostrar los resultados
        System.out.println("La suma de los dos primeros números es: " + suma);
        System.out.println("El producto de los dos últimos números es: " + producto);

        // Cerrar el scanner
        scanner.close();
    }
}
